### PR TITLE
Implemented some gitflow hooks

### PR DESCRIPTION
**Pull request by marcinant**
Support for [GitFlow](https://github.com/petervanderdoes/gitflow-avh) `release` and `feature` hooks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-  "name": "git-guppy",
+    "name": "git-guppy",
+    "version": "v1.2.2",
```
Comment (therealklanni) [inline package.json]: Please undo these changes.

**Comment by therealklanni** [comment]
This is great, thanks for the PR! Mind adding tests to cover the new hooks?

**Comment by marcinant** [comment]
Sure. Unfortunately I'm very busy now. I'm going to add tests by the end of next week.

**Comment by therealklanni** [comment]
No worries, I appreciate your contributions!
